### ISSUE TITLE
getDuration is not a function

### ISSUE THREAD

**Issue by rouvenherzog** [kaltura/mwEmbed#3549]
Hey @OrenMe,

Thank you very much for the recent update. Unfortunately, I run into another error now with the html5 youtube player:

In https://github.com/kaltura/mwEmbed/blob/642a3bb89bccb50b8d86e25a9665358a85e26332/modules/ExternalPlayers/resources/mw.EmbedPlayerYouTube.js#L349 the following error is being thrown;

    Uncaught TypeError: this.getPlayerElement(...).getDuration is not a function
        at HTMLDivElement.setDuration
        at mw.DoubleClick.restorePlayer 
        at mw.DoubleClick.onAdError
        at Do.<anonymous>
        at sf (ima3.js?_=1505811945310:71)
        at P (ima3.js?_=1505811945310:69)
        at Do.h.yd (ima3.js?_=1505811945310:337)
        at sf (ima3.js?_=1505811945310:71)
        at P (ima3.js?_=1505811945310:69)
        at Pn.B (ima3.js?_=1505811945310:282)

You can find examples here: http://www.argovia.ch/shows/countdown (Platz 1, Platz 2 and Platz 3 in the middle of the page). The same error occours on this page as well: http://www.radio24.ch/events/mark-forster-131346703.

Thank you in advance!

**Comment by kaltura-hooks** [comment]
Hi @rouvenherzog,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa \"kaltura*\"
```	
	For deb based systems:
```	
	# dpkg -l \"kaltura-*\"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color \"ERR:\|PHP\|trace\|CRIT\|\[error\]\"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by rouvenherzog** [comment]
ping @OrenMe

**Comment by OrenMe** [comment]
Hi @rouvenherzog thanks, I see it.
This is due to managing error paths.
We load the YouTube iframe player and see we have an ad plugin.
We then go and load the ad cause it would play before the youtube entry.
Ad Vast load fails with empty vast.
We preform ad error cleanup and go on to play the YouTube entry, but the YouTube iframe player hasn't been loaded yet... and crash.
I'll update regarding date.

Thanks!

**Comment by itaykinnrot** [comment]
fixed in v2.64
Thanks,
